### ISSUE TITLE
Automated Circelci tests fail in release-1.6 branch

### ISSUE THREAD

**Issue by ethanandrews** [WHOIGit/ooicgsn-roundabout#197]
@joannekoch I added the release-1.6 branch to the Circleci config.yml file so the tests run on that branch. However, the automated tests fail with a Selenium error now. Can you troubleshoot? Thanks!

```
#!/bin/bash -eo pipefail
docker-compose -f local.yml run --rm tests \
   sh -c "until curl -Is http://localhost:8000; do echo 'sleeping'; sleep 10; done  &&
      google-chrome --version && chromedriver --version &&
      nodejs -v &&
      ./RunAllTests-Chrome-Linux.bat"
Starting project_postgres_1 ... 
Starting project_postgres_1 ... doneStarting project_django_1   ... 
Starting project_django_1   ... done^@^@HTTP/1.0 200 OK
Content-Type: text/html; charset=utf-8
Server-Timing: TimerPanel_utime;dur=349.0829999999998;desc="User CPU time", TimerPanel_stime;dur=96.503;desc="System CPU time", TimerPanel_total;dur=445.5859999999998;desc="Total CPU time", TimerPanel_total_time;dur=450.4849910736084;desc="Elapsed time", SQLPanel_sql_time;dur=3.101825714111328;desc="SQL 4 queries", CachePanel_total_time;dur=0;desc="Cache 0 Calls"
X-Frame-Options: SAMEORIGIN
Vary: Cookie, Accept-Language, Origin
Content-Length: 53175
Content-Language: en
X-Content-Type-Options: nosniff
X-XSS-Protection: 1; mode=block
Set-Cookie:  csrftoken=4MDlqg86bt67b0jotqr1bPBpMtFSHERFUtjRq1ol4yPl9pFfb7FNYhhtVAKbFZDF; expires=Wed, 13 Oct 2021 15:12:48 GMT; HttpOnly; Max-Age=31449600; Path=/; SameSite=Lax
Server: Werkzeug/1.0.1 Python/3.8.6
Date: Wed, 14 Oct 2020 15:12:48 GMT

Google Chrome 86.0.4240.75 
ChromeDriver 85.0.4183.83 (94abc2237ae0c9a4cb5f035431c8adfb94324633-refs/branch-heads/4183@{#1658})
v12.19.0
Running Chrome Tests
Running Add & Edit Locations Test. no such element: Unable to locate element: {"method":"css selector","selector":"*[id="id_name"]"} (Session info: headless chrome=86.0.4240.75) NoSuchElementError: no such element: Unable to locate element: {"method":"css selector","selector":"*[id="id_name"]"} (Session info: headless chrome=86.0.4240.75) at Object.throwDecodedError (/node_modules/selenium-webdriver/lib/error.js:550:15) at parseHttpResponse (/node_modules/selenium-webdriver/lib/http.js:565:13) at Executor.execute (/node_modules/selenium-webdriver/lib/http.js:491:26) at processTicksAndRejections (internal/process/task_queues.js:97:5) at async thenableWebDriverProxy.execute (/node_modules/selenium-webdriver/lib/webdriver.js:700:17) at async toWireValue (/node_modules/selenium-webdriver/lib/webdriver.js:139:15) at async /node_modules/selenium-webdriver/lib/webdriver.js:190:16 at async forEachKey (/node_modules/selenium-webdriver/lib/webdriver.js:184:9) at async convertKeys (/node_modules/selenium-webdriver/lib/webdriver.js:189:3) at async thenableWebDriverProxy.execute (/node_modules/selenium-webdriver/lib/webdriver.js:698:22) Add Edit Locations failed.

Exited with code exit status 1
CircleCI received exit code 1
```

**Comment by joannekoch** [comment]
Hi Ethan,

The fix for this is already on my automated_tests branch. Should I submit a pull request for my branch to be merged to 1.6?

Thanks,
Joanne

**Comment by ethanandrews** [comment]
Yes, please!

**Comment by joannekoch** [comment]
Hi Ethan,

That’s strange. I had 3 successful runs on my branch today in circleci.

I’m running yours again to see if it’s a timing issue. There hasn’t been a code change on your branch that no longer allows 2 locations with the same name?

Also, do you know if Sidney is working today. I sent her 2 emails but I haven’t heard back from her.

Joanne

**Comment by ethanandrews** [comment]
No, no changes in the location model. Merge the 1.6 branch back into your local branch to get all the current code for your local testing.

I don’t know Sidney’s schedule today.

**Comment by joannekoch** [comment]
Hi Ethan,

I merged 1.6 into my branch and built from a clean slate. When I add a location, I get the spinning wheel (see below). The spinning wheel is present on the delete location and adding a part template too.

My tests are failing because everything is running very slow.

Joanne




[cid:image001.jpg@01D6A304.23882550]

**Comment by ethanandrews** [comment]
Ok, interesting. Guess the tests work! I will close this and take a look.
